### ISSUE TITLE
dev-server?

### ISSUE THREAD

**Issue by PKlknr** [PKlknr/svatic#8]
It's simple enough to `npx servor out`. But sometimes those things send several reloads in a row because several files changed.

watch.js knows when it's time to reload. Would be nice if it could just tell a dev-server.

**Comment by PKlknr** [comment]
* servor only livereloads on / (it's made for SPA, so that's understandable), and doesn't expose a reload function. It easily hacked, though: https://github.com/PKlknr/servor/tree/for-svatic
* browser-sync is nice now and then, but it's big.

Here's how to try browser-sync:
```

const devServer = async () => {
  const bs = require('browser-sync').create();
  bs.init({server: destDir, open: false});
  return bs.reload;
};

devServer().then(reload =>
  watch({
    afterBuild: reload,
  }),
);

```

**Comment by PKlknr** [comment]
implemented in 307b39c9d3550ff8f24ffbc98a968a6864c31548
